trestle/common/list_utils.py

```diff
@@ -42,3 +42,11 @@ def is_ordered_sublist(needle: List[str], haystack: List[str]) -> bool:
     needle=['a','b','c'], haystack=['x','y','a','b','z','c'], result = False
     """
     return ' '.join(needle) in ' '.join(haystack)
+
+
+def join_str_to_list_dicts(dict1: Dict, dict2: Dict) -> None:
```

Comment:
Please add tests to cover this function

Reply:
Yes! Test added, bug found and fixed. Very good suggestion. Just having higher level code use was insufficient.